script.js

```diff
@@ -11,7 +13,7 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   navItem.classList.remove('active');
 }));
 
-// pop-up window
+// pop-up window in my see project section done with two javascript method.
 
 function displayProjects() {
```

Comment:
- [optional] when your portfolio is launched your project popup comes up without me clicking on it. please fix that